app/script/ui/ContextMenu.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Wire
+ * Copyright (C) 2016 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+'use strict';
+
+window.z = window.z || {};
+window.z.ui = z.ui || {};
+
+z.ui.Context = (() => {
+
+  let onWheel;
+  let onKeyup;
+
+  function addListeners() {
+    window.addEventListener('wheel', onWheel = (e) => e.preventDefault());
+    window.addEventListener('keyup', onKeyup = (e) => {
+      // TODO: arrow keys
+      if (e.keyCode === 27) {
+        cleanup();
+      }
+    });
+    window.addEventListener('click', cleanup);
+    window.addEventListener('resize', cleanup);
+  }
+
+  function removeListeners() {
+    window.removeEventListener('wheel', onWheel);
```

Comment:
Is the order of the removal of `wheel`, `keyup`, etc. important? Otherwise I would sort them:

```javascript
window.removeEventListener('click', cleanup);
window.removeEventListener('keyup', onKeyup);
window.removeEventListener('resize', cleanup);
window.removeEventListener('wheel', onWheel);
```